lib/components/tag-chip/style.scss

```diff
@@ -19,6 +18,14 @@
     }
   }
 
+  &.interactive {
+    cursor: pointer;
+  }
+
+  &.deleted {
+    background: $studio-red-5;
```

Comment:
@SylvesterWilmott based on [the screenshots you shared](https://github.com/Automattic/simplenote-electron/pull/2817#issuecomment-814785727), I used this color for deleted tags in **light theme**, let me know if I should use a different one.

## Screenshot
<img width="744" alt="tag-chip-deleted-light" src="https://user-images.githubusercontent.com/14905380/114052416-555f0280-988e-11eb-9801-4bea6e67591a.png">